xds/internal/xdsclient/bootstrap/bootstrap_test.go

```diff
@@ -990,3 +991,34 @@ func TestNewConfigWithFederation(t *testing.T) {
 		})
 	}
 }
+
+func TestDefaultBundles(t *testing.T) {
+	c := bootstrap.GetCredentials("google_default")
+	if c == nil {
+		t.Errorf(`Bootstrap.GetCredentials("google_default") credential is nil, want non-nil`)
+	}
+
+	c = bootstrap.GetCredentials("insecure")
+	if c == nil {
+		t.Errorf(`Bootstrap.GetCredentials("insecure") credential is nil, want non-nil`)
+	}
+}
+
+func TestCredsBuilders(t *testing.T) {
+	b := &googleDefaultCredsBuilder{}
+	if _, err := b.Build(nil); err != nil {
+		t.Errorf("googleDefaultCredsBuilder Build error = %v, want nil", err)
+	}
+	if got, want := b.Name(), "google_default"; got != want {
+		t.Errorf("googleDefaultCredsBuilder Name = %v, want %v", got, want)
+	}
+
+	i := &insecureCredsBuilder{}
+	if _, err := i.Build(nil); err != nil {
+		t.Errorf("insecureCredsBuilder Build error = %v, want nil", err)
```

Comment:
You could simplify these errors (by skipping the want portion) where you are expecting a `nil` error with something like:
```golang
t.Errorf("insecureCredsBuilder.Build() failed: %v", err)
```

Reply:
Done